src/Microsoft.Azure.WebJobs.Extensions.Kafka/Config/KafkaOptions.cs

```diff
@@ -136,6 +136,23 @@ public int MaxBatchSize
         /// <value>The auto commit interval ms.</value>
         public int AutoCommitIntervalMs { get; set; } = 200;
 
+        /// <summary>
+        /// Gets or sets the debug option for librdkafka library.
+        /// Default = "" (disable)
+        /// A comma-separated list of debug contexts to enable: all,generic,broker,topic,metadata,producer,queue,msg,protocol,cgrp,security,fetch
+        /// Librdkafka: debug
+        /// </summary>
+        public string LibkafkaDebug { get; set; } = null;
+
+        // <summary>
+        // When a topic loses its leader a new metadata request will be enqueued with this
+        // initial interval, exponentially increasing until the topic metadata has been
+        // refreshed. This is used to recover quickly from transitioning leader brokers.
+        // Use this coinfiguration for EventHubs usage. https://github.com/edenhill/librdkafka/issues/3109
+        // default: 250
+        // </summary>
+        public int? TopicMetadataRefreshFastIntervalMs { get; set; }
```

Comment:
Dear @TsuyoshiUshio,

I just wanted to give you a heads up on this.

Within https://github.com/edenhill/librdkafka/issues/3109, there are absolutely no references to `topic.metadata.refresh.fast.interval.ms`.

With kind regards,
Andreas.

Reply:
Removed.